__tests__/e2e/lib/pages/published-page-page.ts

```diff
@@ -33,10 +33,12 @@ export class PublishedPagePage {
         // If text isn't found the first time, reload and check again up to 2 more times.
         let postTry = 0;
         while ( postTry < 3 ) {
+            // eslint-disable-next-line no-await-in-loop
             await this.page.waitForSelector( selectors.entryTitle );
-            if ( await this.page.locator( selectors.pageText( text ) ) ) {
+            if ( this.page.locator( selectors.pageText( text ) ) ) {
```

Comment:
@bsessions85 `this.page.locator()` returns a `Locator` and not a `Promise`; therefore, I removed `await`. According to the `.d.ts` file (and the [documentation](https://playwright.dev/docs/api/class-page#page-locator)), this method cannot return a falsy value. Thus, as far as I understand, this condition is always going to be `true`.

Does it make sense to await for `locator.count()` instead?

Could you please take a look?

Reply:
Oh good question, totally missed that.
lets await for `locator.isVisible()`

Reply:
Done

Reply:
This caused a strict mode violation:

```
    locator.isVisible: Error: strict mode violation: "text=8 Sad Badgers Slap Briefly" resolved to 2 elements:
        1) <a aria-current="page" class="wp-block-pages-list__…>8 Sad Badgers Slap Briefly</a> aka playwright.$("a:has-text("8 Sad Badgers Slap Briefly")")
        2) <h1 class="alignwide wp-block-post-title">8 Sad Badgers Slap Briefly</h1> aka playwright.$("h1:has-text("8 Sad Badgers Slap Briefly")")
```

I replaced `locator.iosVisible()` with `locator.first().isVisible()`.